src/routes/admin/_admin.access-keys.tsx

```diff
@@ -29,7 +31,12 @@ export const Route = createFileRoute('/admin/_admin/access-keys')({
     updatedRange: z.object({ from: z.string(), to: z.string() }).optional().catch(undefined),
     xpubId: z.string().optional().catch(''),
   }),
-
+  errorComponent: ({ error }) => {
```

Comment:
It doesn't seem to work properly with newest spv-wallet 
In case if it will return error or in case if it will be turned off, I'm getting the old fashion (bad looking) error component.

Reply:
Updated ErrorComponent